python/ray/autoscaler/_private/cli_logger.py

```diff
@@ -133,7 +133,9 @@ def _patched_makeRecord(
     This patched version is otherwise identical to the one in the standard
     library.
     """
-    rv = logging.LogRecord(name, level, fn, lno, msg, args, exc_info, func, sinfo)
+    rv = logging._logRecordFactory(
```

Comment:
do you mind adding a note on why we're doing this (tag the github issue) and and a todo to clean it up?

Reply:
The updated line matches the code in the real makedRecord, so I don't think it needs a note (the way it was previously is wrong)

Added a todo to the docstring to remove the hack!